app/src/main/java/com/myperishableplanner/v21001/MainActivity.kt

```diff
@@ -90,9 +91,6 @@ class MainActivity : ComponentActivity() {
         }
     }
 
-    var inItemName = "No item selected"
-    var selecteItem = Item(0, "", "")
```

Comment:
Moved inItemName declaration to the top according with https://kotlinlang.org/docs/coding-conventions.html#class-layout

Removed selecteItem as there is already a selectedItem declared at the top of the class.